src/components/VersionNumber.js

```diff
@@ -9,7 +9,12 @@ import { Wrapper } from './Wrapper.js';
 export const VersionNumber = () => {
   return (
     <Wrapper>
-      <RegularText small style={styles.version}>
+      <RegularText
+        small
+        style={styles.version}
+        accessible={true}
+        accessibilityLabel={`(App version:) ${appJson.expo.version} `}
```

Comment:
i think it is enough to wrap the text in `(...)` or to add a `:` to have a short pause in reading.
there is also a whitespace at the end before closing `, please clean it up.